Backpack/src/main/java/net/skyscanner/backpack/toogle/BpkSwitch.kt

```diff
@@ -0,0 +1,38 @@
+package net.skyscanner.backpack.toogle
```

Comment:
```suggestion
package net.skyscanner.backpack.toggle
```

Will need to be changed across the package

Reply:
🤦‍♂️

Reply:
Done